### DISCUSSION TITLE
Greatest Hits Release filter

### DISCUSSION THREAD

**Discussion by mudfinger** [RandomNinjaAtk/docker-lidarr-extended#177] [General]
Does anyone know how to create a profile to only pull "Greatest Hits" and/or "Best of" releases?  Using "Compilation" works, but I find that I get a ton of other releases for certain Artists.  

What would be really cool is if it were a selectable item in the metadata profile options.  I realize it wouldn't filter out every "greatest hit" release, but would probably work on some/most.

**Comment by Yosh82** [comment]
Not too sure how the LE script would handle this when downloading from Tidal / Deezer, but I believe a release profile would accomplish this (at least for downloads using a nzb / torrent client):

![image](https://user-images.githubusercontent.com/106485860/211169797-49d0ea5d-5e6c-44cf-a265-f80d4f2bd643.png)
